docs/Design/Classes/Member Variables.md

```diff
@@ -119,6 +119,21 @@ this.age;
 this.gender;
 ```
 
+### JavaScript
+
+```javascript
+public name: string;
+protected age: number;
+private gender: string;
```

Comment:
JavaScript doesn't specify member variables. It's more like Python/Ruby in that regard.